modulegen/internal/main.go

```diff
@@ -14,25 +14,25 @@ import (
 	"github.com/testcontainers/testcontainers-go/modulegen/internal/workflow"
 )
 
-func Generate(exampleVar context.ExampleVar, isModule bool) error {
+func Generate(moduleVar context.TestcontainersModuleVar, isModule bool) error {
 	ctx, err := context.GetRootContext()
 	if err != nil {
 		return fmt.Errorf(">> could not get the root dir: %w", err)
 	}
 
-	example := context.Example{
-		Image:     exampleVar.Image,
+	m := context.TestcontainersModule{
```

Comment:
rename it `module` to be explicit

Reply:
I did not want to create a collision with the package. See https://github.com/testcontainers/testcontainers-go/pull/1568/files#diff-fa6baac23d1c3269416dc19f8301b7051795bb3bf9c98298d9e94c4870acfdbeR11

Reply:
Alright but it's a bit too short, don't you think ?

Reply:
I updated it to `tcModule`